src/components/picker/helpers/usePickerMigrationWarnings.tsx

```diff
@@ -0,0 +1,29 @@
+import {useEffect} from 'react';
+import _ from 'lodash';
+import {LogService} from '../../../services';
+import {PickerProps, PickerModes} from '../types';
+
+type UsePickerMigrationWarnings = Pick<PickerProps, 'value' | 'mode'>;
+
+const usePickerMigrationWarnings = (props: UsePickerMigrationWarnings) => {
+  const {value, mode} = props;
+  useEffect(() => {
+    if (mode === PickerModes.SINGLE && Array.isArray(value)) {
+      LogService.warn('Picker in SINGLE mode cannot accept an array for value');
+    }
+    if (mode === PickerModes.MULTI && !Array.isArray(value)) {
+      LogService.warn('Picker in MULTI mode must accept an array for value');
+    }
+
+    // TODO: this warning should be replaced by the opposite
```

Comment:
This TODO is confusing, please leave only the relevant warning and rewrite the comment if still relevant